docs/articles/streams/integration.md

```diff
@@ -4,30 +4,112 @@ title: Integration
 ---
 
 #Integrating with Actors
```

Comment:
Do you think I should change the other documentation files as well? Should be easy with a simple grep search/replace.